packages/react/src/components/card-analytics/CardAnalytics.tsx

```diff
@@ -18,16 +20,17 @@ const menuItems = [
   },
 ];
 
-export const CardAnalytics = ({ isLoading = false, title, contentClass, children, compact = false }: React.PropsWithChildren<CardProps>) => {
+export const CardAnalytics = ({ isLoading = false, title, contentClass, children, compact = false, additionalHeaderContent, menuVisible = true }: React.PropsWithChildren<CardProps>) => {
   const calculateLoadPanelPosition = (): PositionConfig => ({
     of: `.${contentClass + (compact ? ' .title' : ' .content')}`,
     at: compact ? 'right' : 'center',
   });
 
   return (
     <div className={`card ${contentClass}`}>
-      <Menu visible className='overflow-menu' items={menuItems} />
+      <Menu visible={menuVisible} className='overflow-menu' items={menuItems} />
       <div className='title'>{title}</div>
+      {additionalHeaderContent}
       <div className='content'>{children}</div>
       <LoadPanel container={`.${contentClass}`} position={calculateLoadPanelPosition()} visible={isLoading} showPane={false} width='100%' height={compact ? 50 : 60} maxWidth='100%' />
```

Comment:
Looks like we need to put it on more than one line. Difficult read all props. Maybe add lint rules

Reply:
modified prettier settings to prevent long lines